logger/src/lib.rs

```diff
@@ -95,6 +56,25 @@ impl<D> Logger for Service<D>
 where
     D: Dal + Send + Sync + 'static,
 {
+    async fn store_logs(
+        &self,
+        request: Request<StoreLogsRequest>,
+    ) -> Result<Response<StoreLogsResponse>, Status> {
+        let request = request.into_inner();
+        let logs = request.logs;
+        if !logs.is_empty() {
+            _ = self
+                .logs_tx
+                .broadcast(logs.into_iter().map(Log::from_stored).collect())
+                .await
+                .map_err(|err| {
+                    println!("failed to send log to storage: {}", err);
+                });
```

Comment:
The error is emitted here, is that something we want? Maybe we can handle this gracefully and return it.

Reply:
Done.